examples/step-79/doc/intro.dox

```diff
@@ -448,26 +448,26 @@ While most of the discussion above follows traditional and well-known approaches
  to solving nonlinear optimization problems, it turns out that the problem is
  actually quite difficult to solve in practice. In particular, it is quite
  nonlinear and an important question is not just to find search directions
- $c_\bullet$ as discussed above based on a Newton method, but that one needs to
+ $c_{\{\bullet\}}$ as discussed above based on a Newton method, but that one needs to
  spend quite a lot of attention to how far one wants to go in this direction.
- This is often called ``line search" and comes down to the question of how to
+ This is often called "line search" and comes down to the question of how to
  choose the step length $\alpha_k \in (0,1]$ so that we move from the current
- iterate $x_k$ to the next iterate $x_{k+1}=x_k+\alpha_k x_k$ in as efficient a
+ iterate $\mathbf{x}_k$ to the next iterate $\mathbf{x}_{k+1}=\mathbf{x}_k+\alpha_k \mathbf{x}_k$ in as efficient a
```

Comment:
I "vectorised" the update here, since you're working with a vector equation anyway. Question: shouldn't the displacement update read 
```
\mathbf{x}_{k+1}=\mathbf{x}_k+\alpha_k \Delta \mathbf{x}_k
```
( i.e. the update `\Delta \mathbf{x}_k` is an increment)?